MovieApp/Views/HeaderView.swift

```diff
@@ -0,0 +1,303 @@
+// HeaderView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+/// HeaderView растягиваемый хеадер вьюв с блур эффектом фрактион комлит которого привязан к скролу коллектион вьюв
+final class HeaderView: UICollectionReusableView {
+    // MARK: - Visual components
+
+    lazy var movieCover: UIImageView = {
+        let imageView = UIImageView()
+        imageView.contentMode = .scaleAspectFill
+        return imageView
+    }()
+
+    let gradientView = UIView()
+    let gradient = CAGradientLayer()
+
+    let movieName: UILabel = {
```

Comment:
```suggestion
    let movieNameLabel: UILabel = {
```
дай везде нормальные имена